specs/slider/slider.md

```diff
@@ -0,0 +1,189 @@
+
+*Events*
+- `change` - raise the change event for external parties to be informed of the value change.
+
+*Methods*
+- `onValueChanged` - handling the internal framework call when the value attribute is changed.
+
+*Slots*
+- `background-track` - background area encompassing the track and labels
+- `slider-track` - the horizontal or vertical track along which the thumb slides
+- `thumb` - the control the user drags along the track to change the value of the slider
+
+### Anatomy and Appearance
+**Structure:**
+
+```
+  <div
+    role="slider"
+  >
+    <input type="range" 
+      id="${x => x.id}"
+      min="${x => x.min}"
+      max="${x => x.max}"
+      step="${x => x.step}"
+      value="${x => x.value}"
+    >
+    <slot name="label">
+      <div>${x => x.label}</div>
+    </slot>
+    <slot name="background-track">
+      <div></div>
+    </slot>
+    <slot name="slider-track">
+      <div></div>
+    </slot>
+    <slot name="thumb">
+      <div></div>
+    </slot>
+    <slot></slot>
+  </div>
+```
+
+## Implementation
+
+```
+<fast-slider
+    id="slider1"
+    value={32}
+    min={0}
+    max={44}
+    step={1}
+>
+    <h2 slot="label"><b>Density</b></h2>
+    <div slot="thumb"><img src="..."/></div>
+    <fast-slider-label
+      label="32"
+      show-mark="true"
+    >
+    </fast-slider-label>
+</fast-slider>
+```
+
+---
+
+### Accessibility
+
+*Slider* implements the recommended keyboard navigation scheme described [W3C aria-practices](http://w3c.github.io/aria-practices/examples/slider/multithumb-slider.html).
+
+- aria-valuenow will be set as the slider value changes
+- right and up arrow move the slider in increasing value and left and down arrows move it in decreasing increments
+- *Form Input*
+
+
+### Globalization
+
+*Consider whether the component has any special globalization needs such as:*
+
+- *Special RTL handling*
+Slider should render a mirror view in rtl for example:
+![](./images/slider-rtl.png) 
+
+- Strings passed into the component should already be localized
+
+### Test Plan
+
+While testing is still TBD for our web components, I would expect this to align with the testing strategy and not require any additional test support.
+
+---
+
+# Slider label
+
+## Overview
+
+The `slider-label` component provides a default styled label to be placed inside a `slider` component. Users can choose to not show the mark as well as provide custom label content.
+
+### Use Cases
+
+- *A designer customizing the component.*
+If they use <slider-label> elements those will be styled a specific way internally and the dev can override any part of that style they want.
+
+### API
+
+*Component Name*
+- `slider-label`
+
+*Attributes*
+- `show-mark` - boolean to show/hide the mark. Default is show.
+- `position` - used to position the label on the track relative to the min and max values

```

Comment:
"relative to min and max" doesn't really make sense to me.  Maybe more like "used to position the label at the corresponding value on the track".